staramr/blast/plasmidfinder/PlasmidfinderBlastDatabase.py

```diff
@@ -67,7 +67,7 @@ def get_available_databases(cls) -> list:
         A Class Method to get a list of plasmidfinder databases that are currently supported by staramr.
         :return: The list of database_type currently supported by staramr.
         """
-        return ['gram_positive', 'enterobacteriaceae', 'enterobacteriales']
+        return ['gram_positive', 'enterobacteriales', 'enterobacteriales']
```

Comment:
`enterobacteriales` is already in this list, so you just have to remove `enterobacteriaceae` rather than change the text.

Reply:
Haha. Wooops.